bitnami/redis-cluster/templates/redis-statefulset.yaml

```
@@ -373,8 +377,8 @@ spec:
           requests:
             storage: {{ .Values.persistence.size | quote }}
         {{- include "redis-cluster.storageClass" . | nindent 8 }}
-        selector:
         {{- if .Values.persistence.matchLabels }}
+        selector:
```

With this change, `matchExpressions:` could be orphan, right? What about the following?

```suggestion
      {{- if or .Values.persistence.matchLabels .Values.persistence.matchExpressions }}
        selector:
        {{- if .Values.persistence.matchLabels }}
```

---

You will need to add an `{{- end }}` below, to close the new conditional.